experimental/generation/generator/src/commands/dialog/integrate.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
Do we still need this command?  I'm wondering when someone would want to integrate, but not regenerate.

Reply:
I'd like to keep the integrate command. I think sometimes user might need to merge two existing assets, and this command could be useful.